test/common/performance/aggregator/aggregator.go

```diff
@@ -267,14 +280,14 @@ func publishThpt(timestamps []time.Time, q *quickstore.Quickstore, metricName st
 }
 
 // waitForEvents blocks until the expected number of events records has been received.
-func (ex *aggregatorExecutor) waitForEvents() {
-	for receivedRecords := uint(0); receivedRecords < expectRecords; receivedRecords++ {
+func (ex *Aggregator) waitForEvents() {
```

Comment:
Nit: I find it a bit confusing to see a var named `ex` refer to a type `Aggregator`